riscv-asm.md

```diff
@@ -426,6 +426,36 @@ msg:
 	    .string "Hello World\n"
 ```
 
+Function Calls
+--------------
+
+The following pseudo instructions are available to call subroutines far from
+the current position:
+
+  * `call	<symbol>`: call away subroutine
+  * `tail	<symbol>`: tail call away subroutine
```

Comment:
We also support "jump symbol,rt".  tail clobbers t1 which is OK because it can only occur at the end of a function.  Jump is the same operation as tail, but lets you specify what register to use for the temporary.